Dockerfile

```diff
@@ -5,25 +5,26 @@ FROM golang:$VERSION
 ARG AWS_S3_MOUNT_DIRECTORY=/mnt/s3
 
 # Tools
-# validate s3fs-fuse with the sec team
 RUN apt-get -qq update && apt-get -qq install -y \
     make \
     curl \
+    wget \
     gnupg2 \
     bzip2 \
     createrepo \
-    unzip
+    unzip \
+    golang \
```

Comment:
why do we need go to install?

Reply:
leftover from previous tests on different images, removed